src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -93,6 +93,8 @@ private void testGeneralQuestionOperations() throws Exception {
 
         testEditQuestionLink();
         testEditQuestionAction();
+        
+        testToggleFeedbackPathDisplayLink();
```

Comment:
Maybe standardize naming with the `clickToggleCustomFeedbackPath...` below and add the word `custom`